spring-vault-core/src/main/java/org/springframework/vault/core/VaultKeyValueMetadataTemplate.java

```diff
@@ -0,0 +1,82 @@
+package org.springframework.vault.core;
+
+import java.io.IOException;
+
+import org.springframework.util.Assert;
+import org.springframework.vault.VaultException;
+import org.springframework.vault.client.VaultResponses;
+import org.springframework.vault.support.VaultMetadataRequest;
+import org.springframework.vault.support.VaultMetadataResponse;
+import org.springframework.vault.support.VaultMetadataResponseWrapper;
+import org.springframework.web.client.HttpStatusCodeException;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+public class VaultKeyValueMetadataTemplate implements VaultKeyValueMetadataOperations {
+
+  private final VaultOperations vaultOperations;
+
+  private static final ObjectMapper OBJECT_MAPPER;
+
+  static {
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
Given the need for another dependency, we should rather parse JSR-310 dates ourselves (see `VaultVersionedKeyValueTemplate.getDate(…)`)